js/views/modals/About/Contributors.js

```diff
@@ -1,35 +1,302 @@
 import loadTemplate from '../../../utils/loadTemplate';
 import baseVw from '../../baseVw';
+import regeneratorRuntime from 'regenerator-runtime';
 
-const names = [
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
-  'Zach Galifianakis',
-  'LeBron James',
+let names;
+let fetchNames;
+
+// Linter notes:
+  // linter will not be happy with some things which also work:
+  // no-use-before-define -- because of const variable hoisting and invocation time
+  // this works and is a way to share data in closures. The linter is also unhappy.
```

Comment:
If we're doing things to get around the linter, there's probably a problem with what we're doing.